webpack.common.js

```diff
@@ -7,12 +7,20 @@ const webpack = require('webpack');
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 const {VueLoaderPlugin} = require('vue-loader');
-const gitRevision = require('child_process')
-    .execSync('git rev-parse HEAD')
-    .toString().trim();
-const gitBranch = require('child_process')
-    .execSync('git rev-parse --abbrev-ref HEAD')
-    .toString().trim();
+
+let gitRevision = 'error-retrieving-revision';
+let gitBranch = 'error-retrieving-branch';
```

Comment:
how would this appear in the about modal?

Reply:
Just like that, instead of the branch and hash it would be these strings letting you know there was an issue. The plus side is, that the build will complete though. I'm looking into other ways of getting this information if possible.

Reply:
Actually... looks like this was merged in elsewhere at some point, so this PR is moot.